### ISSUE TITLE
About the training protocol

### ISSUE THREAD

**Issue by isCopyman** [yandex-research/tabred#3]
After dividing the training, validation and test sets, searching for hyperparameters and early stops on the validation set, does this mean that the validation set was not finally added to training data to retrain a model?

**Comment by puhsu** [comment]
Yes, the validation set was used just for tuning and early stopping. We do not add it to the training data (for one, we should employ a different stopping strategy if we do not have the validation set). 


_Slight off top:_
You might be interested in the follow-up experiments with splitting strategies on tabred datasets in https://arxiv.org/abs/2502.20260 (I do find some results intriguing but still need to digest and double check).

**Comment by isCopyman** [comment]
Yes, I just read the paper https://arxiv.org/abs/2502.20260  and it gives some counterintuitive results. Interesting.
